src/js/tech/middleware.js

```diff
@@ -0,0 +1,70 @@
+import Tech from './tech.js';
+import toTitleCase from '../utils/to-title-case.js';
+import assign from 'object.assign';
+
+const middlewares = {};
+
+export function use(type, middleware) {
+  (middlewares[type] = middlewares[type] || []).push(middleware);
+}
+
+export function setSource(src, next) {
+  setTimeout(()=>ssh(src, middlewares[src.type] || [], next, []), 1);
```

Comment:
I thought Standard required spaces here (i.e. `() => foo()`)?

I don't feel crazy strong either way, but on [line 16](https://github.com/videojs/video.js/pull/3788/files#diff-b1bec4e9f074fe980f0471fcaaf4df75R16) there are spaces so we're not even being consistent within the same file.